docs/embedding/sdk/upgrade.md

```diff
@@ -7,13 +7,13 @@ summary: How to upgrade your Metabase and Embedded analytics SDK versions, test
 
 Here's a basic overview of the steps you'll want to take when upgrading your SDK.
 
-## 1. Read the release post and changelog for both Metabase and the Embedded analytics SDK
+## 1. Read the release post and changelog for Metabase and the Embedded analytics SDK
 
 - [Release posts](https://www.metabase.com/releases) give a good overview of what's in each release, and call out breaking changes (which are rare).
 - [Metabase changelogs](https://www.metabase.com/changelog) list Metabase changes.
-- [Embedded analytics SDK changelogs](https://www.metabase.com/changelog/55) list changes specific to the SDK package.
+- [Embedded analytics SDK changelogs](https://www.metabase.com/changelog/56) list changes specific to the SDK Package.
 
-Check for any relevant changes, especially breaking changes that require you to update your application's code. If there are breaking changes, we'll have docs that'll walk you through what changes you'll need to make and why.
+Check for any relevant changes, especially deprecation changes that require you to update your application's code. If there are deprecation changes, we'll have docs that'll walk you through what changes you'll need to make and why.
```

Comment:
Why deprecation vs breaking? Are we claiming there will never be breaking changes?

Reply:
I would put "deprecations or breaking changes" to cover both grounds; there will still be breaking changes but it will be much rarer than before.

Reply:
Starting from 58 with Hosted Bundle we should not have breaking changes between 2 sibling versions in a public API. We deprecate something first, so consumers will have 2 months to update their code. So technically it's not a breaking change.